pyramid_zipkin/request_helper.py

```diff
@@ -119,13 +119,22 @@ def create_zipkin_attr(request):
     Attaches lazy attribute `zipkin_trace_id` with request which is then used
     throughout the tween.
 
+    consumes custom is_tracing function to determine if the request is traced
+    if one is set in the pyramid registry
+
     :param request: pyramid request object
     :rtype: :class:`pyramid_zipkin.request_helper.ZipkinAttrs`
     """
     request.set_property(get_trace_id, 'zipkin_trace_id', reify=True)
 
     trace_id = request.zipkin_trace_id
-    is_sampled = is_tracing(request)
+
+    settings = request.registry.settings
+    if 'zipkin.is_tracing' in settings:
+        is_sampled = settings['zipkin.is_tracing'](request)
+    else:
+        is_sampled = is_tracing(request)
```

Comment:
@lorinbeer [Monkey patching](https://en.wikipedia.org/wiki/Monkey_patch) is the way to do that in python

@mjbryant I think we are effectively monkey patching, but not explicitly. I think i'd rather have something like an extension module that we can load at runtime. That module would contain a subclass of some plugin base class that we define. That way the interface is more strictly defined.

Anyway this is out of scope of this PR, but something to think about down the road.